xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -40,11 +41,23 @@
 #include "utils/URIUtils.h"
 #include "addons/AddonManager.h"
 #include "addons/InputStream.h"
+#include "Util.h"
 
 
-CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IVideoPlayer* pPlayer, CFileItem fileitem)
+CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IVideoPlayer* pPlayer, CFileItem fileitem, bool scanforextaudio)
 {
   std::string file = fileitem.GetPath();
+  if (scanforextaudio)
```

Comment:
you seem to be concerned about performance. I wondering if this should be optional and activated by a setting.